en/docs/setup/deployment-patterns.md

```diff
@@ -1,20 +1,23 @@
 # Deployment Patterns
 
 The following sections provide high level information on
-the recommended deployment pattern. 
+the recommended deployment patterns. 
```

Comment:
There is only one deployment pattern.

Reply:
Fixed: d2ff3de28eeaeeecdeea77613853ab5cd9e5cd23